src/unix/notbsd/linux/musl/mod.rs

```diff
@@ -0,0 +1,131 @@
+pub type clock_t = ::c_long;
+
+pub type blkcnt_t = i64;
+pub type fsblkcnt_t = u64;
+pub type fsfilcnt_t = u64;
+pub type ino_t = u64;
+pub type off_t = i64;
+pub type stat64 = ::stat;
```

Comment:
The differences between stat and stat64 are two fields that changed from ino_t to ino64_t and blkcnt_t to blkcnt64_t. But in musl, for any arch, ino_t is always equal to ino64_t; same thing with blkcnt_t.